components/schema-migrator/migrations/1_initialize_schema.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE runtime_status_condition AS ENUM ('INITIAL', 'READY', 'FAILED');
+
+CREATE TABLE runtime
+(
+    id uuid NOT NULL CONSTRAINT runtime_pk PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    name varchar(256) NOT NULL,
+    description text,
+    status_condition runtime_status_condition DEFAULT 'INITIAL'::runtime_status_condition NOT NULL,
+    status_timestamp timestamp NOT NULL,
+    auth json
+);
+
+CREATE UNIQUE INDEX runtime_id_uindex ON runtime (id);
+CREATE UNIQUE INDEX runtime_id_name_uindex ON runtime (tenant_id, name);
```

Comment:
why do we have the second index `runtime_id_name_uindex`?
In GraphQLAPI we query by runtime ID:
```
runtime(id: ID!): Runtime
```